WordPress/twentytwentyfour
# Improve color palette design system

**Description**
Organizes, cleans-up, and structures colors in support of a variable design system that can be flexed/reduced better. 

#### Key concepts: 

1. Use "Base / $variant" structure for naming colors, instead of actual color representation. Example: "Accent / Three" instead of "Brick". This way users can edit the color in Global Styles and it won't always render the color title as "Brick". The `/ $variant` notation helps attribute this color as an intended alternate. 

2. Uses numeral values, instead of primary/secondary/tertiary/etc, as those are indicative of priority, instead of alternatives. 

3. Reduces and consolidates color values (matching the initial Figma designs). Theme variations should register alternates  instead of including those colors in the parent. 


**Screenshots**

| Before  | After |
| ------------- | ------------- |
|<img width="262" alt="CleanShot 2023-08-28 at 11 47 12" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/437b5520-ef4e-4f0b-a4b5-c9a99138925e">|<img width="262" alt="CleanShot 2023-08-28 at 11 51 33" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/4b308238-ab29-4e1e-a1bb-b8b219c92ddb">|

CSS:
<img width="293" alt="CleanShot 2023-08-28 at 12 24 24" src="https://github.com/WordPress/twentytwentyfour/assets/1813435/b7cce399-c8c9-4254-ae04-a94e2ab2f204">

**Testing Instructions**
1. View the color settings of any block that supports color. See the new palette.
